_fluo-1-2/administration/run-fluo-in-docker.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Run Fluo in Docker
+category: administration
+order: 4
+---
+
+## Obtain the Docker image
+
+To obtain the docker image created by this project, you can either pull it from DockerHub at
+`apache/fluo` or build it yourself. To pull the image from DockerHub, run the command below:
+
+    docker pull apache/fluo
+
+While it is easier to pull from DockerHub, the image will default to the software versions below:
+
+| Software    | Version |
+|-------------|---------|
+| [Fluo]      | 1.2.0   |
+| [Accumulo]  | 1.8.1   |
+| [Hadoop]    | 2.7.3   |
+| [Zookeeper] | 3.4.9   |
+
+If these versions do not match what is running on your cluster, you should consider building
+your own image with matching versions. However, Fluo must be 1.2+. Below are instructions for
+building an image:
+
+1. Clone the Fluo docker repo
+
+        git clone git@github.com:apache/fluo-docker.git
+
+2. Until Fluo 1.2 is released, build a Fluo tarball distribution and copy it to the root
+   directory of the repo.
+
+        git clone git@github.com:apache/fluo.git
+        cd fluo/
+        mvn clean package
+        cp modules/distribution/target/fluo-1.2.0-SNAPSHOT-bin.tar.gz /path/to/fluo-docker/
+
+3. Build the default Fluo docker image using the command below.
+
+        cd /path/to/fluo-docker
+        docker build -t fluo .
+
+   Or build the Fluo docker image with specific versions of Hadoop, Zookeeper, etc using the command below:
+
+        docker build --build-arg ZOOKEEPER_VERSION=3.4.8 --build-arg ACCUMULO_VERSION=1.7.3 --build-arg HADOOP_VERSION=2.7.0 -t fluo .
+
+## Image basics
+
+The entrypoint for the Fluo docker image is the `fluo` script. While the primary use
+case for this image is to start an oracle or worker, you can run other commands in the
+`fluo` script to test out the image:
+
+```bash
+# No arguments prints Fluo command usage
+docker run apache/fluo
+# Print Fluo version
+docker run apache/fluo version
+# Print Fluo classpath
+docker run apache/fluo classpath
+```
+
+## Run application in Docker
+
+Before starting a Fluo oracle and worker using the Fluo Docker image, [initialize your Fluo application][initialize]. 
+Next, choose a method below to run the oracle and worker(s) of your Fluo application. In the examples below, the Fluo
+application is named `myapp` and was initialized using a Zookeeper node on `zkhost`.
+
+### Docker engine
+
+Use the `docker` command to start local docker containers.
+
+1. Start a Fluo oracle
+
+        docker run -d --network="host" apache/fluo oracle -a myapp -o fluo.connection.zookeepers=zkhost/fluo
```

Comment:
could give a one sentence description of what each argument and option does.

Reply:
Fixed